webapp/templates/team/submit_modal.html.twig

```diff
@@ -80,4 +123,31 @@
             filesSelected.removeClass('d-none');
         });
     </script>
+    <style>
+        .container {
```

Comment:
Can you add this to the `.ccs` files and not directly in the page?

Reply:
Ok